app/code/Magento/Review/Block/Adminhtml/Add.php

```diff
@@ -28,16 +28,10 @@ protected function _construct()
 
         $this->buttonList->update('save', 'label', __('Save Review'));
         $this->buttonList->update('save', 'id', 'save_button');
+        $this->buttonList->update('save', 'class', 'save primary no-display-custom');
 
-        $this->buttonList->update('reset', 'id', 'reset_button');
-
-        $this->_formScripts[] = '
-            require(["prototype"], function(){
-                toggleParentVis("add_review_form");
-                toggleVis("save_button");
-                toggleVis("reset_button");
-            });
-        ';
+        $this->buttonList->update('reset', 'id', 'reset_button');    
+        $this->buttonList->update('reset', 'class', 'reset no-display-custom');
```

Comment:
Please avoid usage of such generic names as "custom", because it has no implication or significance.
The name of the CSS class should be self-explanatory.